packages/fluentui/react-builder/src/components/List.tsx

```diff
@@ -37,6 +37,74 @@ export const List: React.FunctionComponent<ListProps> = ({ onDragStart, style })
     },
   );
 
+  const titleRenderer = (Component, { content, expanded, open, hasSubtree, ...restProps }) => (
```

Comment:
I would probably avoid using `titleRenderer` I think in the future we want to deprecate it, you can achieve the same using `as` prop in the `addComponentToGroup` method:

```javascript
  const addComponentToGroup = (info, property) => {
    componentGroupItems[property].push({
      id: info.displayName,
      as: ({ content, expanded, open, hasSubtree, children, ...restProps }) => {
        return (
          <>
            {hasSubtree ? expanded ? <TriangleDownIcon /> : <TriangleEndIcon /> : null}
            {children}
          </>
        );
      },
```